kolibri/core/sqlite/management/commands/vacuumsqlite.py

```diff
@@ -0,0 +1,49 @@
+import logging as logger
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db import DEFAULT_DB_ALIAS
+
+logging = logger.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Vacuum Kolibri's SQLite database to optimize it and reduce the .wal file size"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--database', action='store', dest='database', default=DEFAULT_DB_ALIAS,
+            help='Specifies the database to vacuum. Defaults to the "default" database.',
+        )
+        parser.add_argument(
+            '--interval', action='store', dest='interval', default=0, type=int,
+            help='Specifies the interval (in minutes) to run the process continuosly. If 0, no repetition will happen',
+        )
```

Comment:
Can we remove the interval+sleep thing?

While I think that it will be okay when running `kolibri stop`, I'm unsure if this might end up blocking other operations, leave unclosed db connections, PID files etc. Certainly, if integrating with `iceqube`, it would require a well-defined behavior for "eternal" threads and how they are shut down. For some purposes, `iceqube` should wait for a thread to finish nicely, and for others, it shouldn't.

The easiest solution is to 1) not run a loop but just execute `VACUUM` once and 2) avoid integrating system maintenance in `iceqube`, because it's not intended for this.

Reply:
I don't understand your objections. Also I did initially only on start and other people requested to have it run periodically to ensure long-running servers are optimized too. If, as Jamie said in slack, our users don't run Kolibri as a server for long periods of time, this does not matter. Anyway, I think this is a discussion that should be held in Slack between the intersted people.

Reply:
Sorry, I have been confused by the many inputs on Slack. At first, I thought it was a Cron job, then I saw @indirectlylit 's comment about making it an Iceqube job, and now I see that it's a daemonized thread.